Wikipedia/assets/codemirror/codemirror-index.html

```diff
@@ -407,8 +407,18 @@
           }, 25)
         }
       })
+      editor.on('change', (doc) => {
```

Comment:
hmm i think onchange might be too expensive as it will happen every time a character is typed... 🤔 can we just call sendNativeMessages once when the find/replace interface is dismissed?

Reply:
it's called then but it needs to be called in between undos

Reply:
maybe I'll just call it after each undo

Reply:
oops wrong bug 🤣

in that case may want to just call it after the undo call?